FileHandler.gd

```diff
@@ -16,3 +16,51 @@ static func create_directory_if_missing(path: String):
 		var success = dir.make_dir(path)
 		if success != OK:
 			print("Error: Could not create data folder:", path)
+
+static func find_file_duplicate(directory_path: String, target_name: String, target_content: PoolByteArray, relative_path: String):
+	var dir = Directory.new()
+	if dir.open(directory_path) == OK:
+		dir.list_dir_begin(true)
+		var file_name = dir.get_next()
+		while file_name != "":
+			var full_path = directory_path.plus_file(file_name)
+			var current_relative_path = relative_path.plus_file(file_name)
+			if dir.current_is_dir():
+				var found_path = find_file_duplicate(full_path, target_name, target_content, current_relative_path)
+				if found_path != "":
+					dir.list_dir_end()
+					return found_path
+			if file_name == target_name:
+				var file = File.new()
+				file.open(full_path, File.READ)
+				var file_content = file.get_buffer(file.get_len())
+				file.close()
+				if file_content == target_content:
+					dir.list_dir_end()
+					return current_relative_path
+			file_name = dir.get_next()
+		return ""
```

Comment:
return null from this function when a file is not found instead of an empty string.